R/nhs_selectizeInput.R

```diff
@@ -1,36 +1,54 @@
-#' nhs_selectizeInput Function
+#' nhs_selectizeInput
 #'
-#' @importFrom shiny tagList
-nhs_selectizeInput <- function(inputId,
+#' NHS styled selectizeInput
+#'
+#' @param inputId Element id
+#' @param label Element label
+#' @param full_width Boolean, controls if full width or not
+#' @param ... Further named args passed to \code{shiny::selectInput}
+#'
+#' @return HTML
+#' @export
+#'
+#' @examples
+#' nhs_selectizeInput(
+#'   "fruit",
+#'   "Choose some fruit",
+#'   full_width = TRUE
+#' )
+nhs_selectizeInput <- function(inputId, # Exclude Linting
                                label,
-                               full_width = FALSE) {
+                               full_width = FALSE,
+                               ...) {
   # Create selectize input
-  nhs_selectizeInput <- shiny::selectizeInput(
+  nhs_selectizeInput <- selectizeInput( # Exclude Linting
```

Comment:
this one hasn't given a name like others? (nsi etc)

Reply:
I think I originally deleted this file, because it seems the function is not doing anything. Then decided to keep it anyway, but didn't change anything.